addons/website/static/src/scss/website.ui.scss

```diff
@@ -527,3 +527,8 @@ body .modal {
         max-height: 14.2rem;
     }
 }
+
+// Post Submit Links
+.post_link:not(.o_post_link_js_loaded) {
+    pointer-events: none;
```

Comment:
Mmh, kinda duplicating the `o_wait_lazy_js` system but as:
- this handles all post_link generically
- requires only a CSS update
- this makes the link clickable at the exact right time (while there would still be a 1ms window where it would break with `o_wait_lazy_js`)

I guess it's fine :+1:

Reply:
I'll use that in master :+1: